src/components/edit-exercise.component.js

```diff
@@ -99,7 +99,7 @@ export default class EditExercise extends Component {
     return (
       <div>
         <h3>Edit Exercise ;pg</h3>
-        <form onSubmit={this.onSubmit}>
+        <form onSubmit={this.onsubmit}>
```

Comment:
could have given the name of the method in camelCase. Otherwise the code looks fine